### DISCUSSION TITLE
Movable Judgement Text

### DISCUSSION THREAD

**Discussion by FundamentalSequence** [ppy/osu#26860] [Ideas]
I'm looking to do a simple task: move the judgement text that appears upon hitting a note in osu!mania higher up the screen:
![osu_2024-01-30_20-38-38](https://github.com/ppy/osu/assets/19355183/7ce30e44-977c-4f67-9f05-09de63f4ddda)
I've done similar fiddling on classic osu!mania in years past, so I thought this would be straightforward. However, this currently appears to be impossible, and I'm not seeing any information on the relevant fora that would have me believe otherwise.

Are there plans to make gameplay elements like this accessible for editing?

**Comment by Rekunan** [comment]
To answer the question, yes, eventually all elements will be accessible by the skin editor. The question of when will be answered sometime in the future.
